lib/python/qmk/cli/doctor/check.py

```diff
@@ -18,6 +17,18 @@ class CheckStatus(Enum):
 
 
 ESSENTIAL_BINARIES = {
+    'make': {
+        'version_arg': '--version'
+    },
+    'git': {
+        'version_arg': '--version'
+    },
+    'dos2unix': {
+        'version_arg': '--version'
+    },
+    'diff': {
+        'version_arg': '--version'
+    },
```

Comment:
`version_arg` defaults to `--version`
```suggestion
    'make': {},
    'git': {},
    'dos2unix': {},
    'diff': {},
```